brian2/tests/test_synapses.py

```diff
@@ -2264,4 +2280,5 @@ def test_synapses_refractory_rand():
     test_synapse_generator_random_with_condition()
     test_synapses_refractory()
     test_synapses_refractory_rand()
+    test_synapse_generator_range_noint()
```

Comment:
Almost there: the function is called ` test_range_noint_error` and not `test_synapse_generator_range_noint()` (but I'd actually prefer the latter name) :smile: